### ISSUE TITLE
Linux mirror fails regardless of what is selected

### ISSUE THREAD

**Issue by ineuw** [linuxmint/linuxmint#499]
```
 * Cinnamon 5.2.7
 * Distribution - Mint 20.3
 * Graphics Intel Graphics native driver
 * 64 bit
 * Attached 220629_xsession-errors.txt, Screenshot from 2022-06-29 12-51-59
 *  
```
Regardless of the mirror selected for update, the message (attached) appears at all times. Refreshing corrects this. I always had this issue intermittently. I recently increased the bandwidth and upgraded the internet service. But the problem remains.

![Screenshot from 2022-06-29 12-51-59](https://user-images.githubusercontent.com/10367016/176511364-842e2c2c-e1bf-40fb-888a-ab08ed25cd8a.jpg)

[220629_xsession-errors.txt](https://github.com/linuxmint/cinnamon/files/9013214/220629_xsession-errors.txt)
![Screenshot from 2022-06-29 12-51-59](https://user-images.githubusercontent.com/10367016/176511512-b8e872df-0249-44e0-a727-fefe1ec06114.jpg)

**Comment by JosephMcc** [comment]
Moved this issue to where it belongs. It's not a Cinnamon issue.

**Comment by ineuw** [comment]
@Tibladar, it happened again just now. Please see the attachments. 

@JosephMcc If this is not a Cinnamon issue, can you tell me what is it and where you moved it please? 

[apt-cache_policy.txt](https://github.com/linuxmint/linuxmint/files/9043131/apt-cache_policy.txt)
![Screenshot from 2022-07-05 00-01-34](https://user-images.githubusercontent.com/10367016/177247478-4dace5c9-3a00-4515-92fb-8e1a5618b480.jpg)

**Comment by JosephMcc** [comment]
I moved it under linuxmint/linuxmint which is general Mint issues. You can see it at the top of the page. Everything gets moved when the issue is moved. You can see your whole thread is now here. You might get more help troubleshooting this on the Mint forums.

**Comment by Tibladar** [comment]
@ineuw Your apt-cache policy looks fine, so that's not the problem.

When the error occurs, does the following command in a bash return anything?
`echo $MINTUPDATE_TEST`

**Comment by ineuw** [comment]
@JosephMcc, your explanation is much appreciated. @Tibladar, there never was ```echo $MINTUPDATE_TEST``` 

This may be a possible source of the issue, because the Update Missing keys in Sources Maintenance keeps re-updating the same keys after initial software installation. I don't know if this is important, but perhaps you can point me to the proper cinnamon site to post this.

I started over with Timeshift backup copy of the June 24 fresh install, completely updated at that date, but without any user added software or modifications. I assumed that once a software is registered with a key, it is a lasting occurrence, and no additional update needed.

The first image is when software installation triggered the sources menu. I know that a missing key update was required at that point to proceed and it's a standard procedure. 

![220704_14-39_missing_keys_on_original_installation](https://user-images.githubusercontent.com/10367016/177450436-3ab702d8-5ead-4577-b30a-97e8ce0f61bb.jpg)

The second image was taken when on a spur of the moment I activated the missing keys in Maintenance.

![220705_13-24_missing_keys](https://user-images.githubusercontent.com/10367016/177450448-4801936b-8ea7-46da-93e8-553d4d3a7a05.jpg)

**Comment by Tibladar** [comment]
@ineuw 
There shouldn't be a variable named `MINTUPDATE_TEST` as it's ment for testing only and triggers the APT corrupt error.
If it's not existing it's right.

I'm out of clues why you're APT configuration is corrupt because those where the two things that could trigger the error:
- Missing Linux Mint entry in apt-cache policy (The file you uploaded is fine)
- MINTUPDATE_TEST is set

**Comment by ineuw** [comment]
Thanks again. I omitted mentioning the last possibility, because it's only my supposition. Wine 7.0 installation is not functioning properly, because it's not installed properly by Wine, (not Mint), whatever method of source and installation I used, (same version). But, WineHQ forum support was unresponsive to my detailed posts. I was hoping that something provided here, by the gods of Mount Cinnamon (in reference to Mount Olympus), would lead to post more detail on the wine forum.
